### PR TITLE
Try/style authenticator

### PR DESCRIPTION
**Pull request by aerych**
Refs #83 

This PR is a quick touch up to the Authenticator styles. Still a WIP as we're lacking a proper navbar icon and it looks like the colors of title/subtitle lables on the username/password screen can not be configured for dark mode. We'll revisit again in a future update after unified login lands in the auth pod.  For now, we're just making sure NiOS stays relatively current with changes.

@mindgraffiti would you be game for a quick review of the changes and a spot check of light/dark mode of the screens?  Any/all feedback welcome!

### CODE REVIEW

File: Newspack/Newspack/Styles/UIColor+SemanticColors.swift [hunk 1]
```diff
@@ -75,11 +75,7 @@ extension UIColor {
     /// Text. Gray-80 (< iOS 13) and `UIColor.label` (> iOS 13)
     ///
     static var text: UIColor {
-        if #available(iOS 13, *) {
-            return .label
-        }
-
-        return .gray(.shade80)
```
Comment (mindgraffiti) [inline Newspack/Newspack/Styles/UIColor+SemanticColors.swift:82]: Are you supporting only iOS 13 & up now?

**Review by mindgraffiti — APPROVED** [review]
Very little is changing on the old UI at the moment. If you're feeling adventurous, update to 1.22.0-beta, which contains icon color changes for the right-hand icons on the username / password fields.

ETA for unified flows is late August.

1️⃣ question
:shipit: 

| This | That | the other thing |
| --- | --- | --- |
| <a href="https://user-images.githubusercontent.com/1062444/88972639-7c832000-d27b-11ea-934f-28546ffb9a57.png"></a> | <a href="https://user-images.githubusercontent.com/1062444/88972669-86a51e80-d27b-11ea-806d-84c742a918e6.png"></a> | <a href="https://user-images.githubusercontent.com/1062444/88972691-902e8680-d27b-11ea-9348-d026dff691a0.png"></a> |

File: Newspack/Newspack/Styles/UIColor+SemanticColors.swift (hunk 1 above)
Comment (aerych) [reply Newspack/Newspack/Styles/UIColor+SemanticColors.swift:82]: Yep! iOS 13 from the get go, and I'll probably switch to iOS 14 and up before launch assuming all dependencies are compliant.

**Comment by aerych** [comment]
Thank you @mindgraffiti !!

**Pull request merged**